lib/justifi/payment.rb

```diff
@@ -25,5 +25,15 @@ def self.create(params: {}, headers: {}, idempotency_key: nil)
         headers: headers,
         idempotency_key: idempotency_key)
     end
+
+    def self.create_refund(amount:, reason:, payment_id:, description: nil)
+      raise ArgumentError, "Refund reason must be one of the following: #{Justifi::REFUND_REASONS}" unless Justifi::REFUND_REASONS.include? reason
```

Comment:
shouldn't we just let the API do this? This makes it another place we have to change it if the api changes.